App.js

```diff
@@ -1,25 +1,29 @@
 import React, { useState, useEffect } from 'react';
 import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
-import SvgComponent from "./LogoSvg";
+import LogoSvg from "./LogoSvg";
+import Quotes from "./Quotes";
+import AnimateSvg from './AnimateSvg';
+import AnimateQuotes from './AnimateQuotes';
 
 export default function App() {
-
   return (
-    <View style={styles.container}>
-      <SvgComponent />
+    <View style={styles.window}>
+      <AnimateSvg>
+        <LogoSvg />
+      </AnimateSvg>
+      <AnimateQuotes>
+        <Quotes />
+      </AnimateQuotes>
     </View>
   )
 };
 
 const styles = StyleSheet.create({
-  container: {
+  window: {
```

Comment:
Changed StyleSheet property name, to signify it being the window of the app.

Reply:
`window` keyword has a specific use in web development that usually refers to the whole environment. I would change this back to container